XLS-47d-PriceOracles/README.md

```diff
@@ -186,82 +164,63 @@ The transaction fails if:
 - `PriceDataSeries` has array elements with missing `AssetPrice`.
 - The `Account` account doesn't exist or the `Account` is not equal to the `Owner` field when updating the Oracle instance.
 - The transaction is not signed by the `Account` account or the account's multi signers.
-- The `URI` field length exceeds 64 bytes.
-- The `Provider` field length exceeds 64 bytes.
+- The `URI` field length exceeds 32 bytes.
+- The `Provider` field length exceeds 32 bytes.
```

Comment:
There are mentions of 256 bytes in the fields list in the section titled "The `PriceOracle` Object". (I cannot make this comment on those lines in this PR because they were not changed.)